ui/client/src/dialogs/ReverseKeyLookup.tsx

```diff
@@ -87,9 +89,26 @@ export const ReverseKeyLookupDialog: React.FC<Props> = ({
         const index = path.lastIndexOf('.');
         if(index !== -1) {
           setParent(path.substring(0, index));
-          setFilter(path.substring(index + 1))
+          // setFilter(path.substring(index + 1))
```

Comment:
still need this?

Reply:
Great point, this was part of a refactoring where the filter was changed for a completely different function. I forgot to remove the commented out invocation of the previous version. Will address it.